daemon/pod.go

```diff
@@ -42,11 +43,201 @@ type Pod struct {
 	sync.RWMutex
 }
 
-func NewPod(rawSpec []byte, id string, data interface{}) (*Pod, error) {
+// TODO: remove convertToRunvPodSpec after pod.UserPod is deleted from runv
+func convertToRunvPodSpec(podSpec *apitypes.UserPod) (*pod.UserPod, error) {
```

Comment:
it is not RunvPodSpec. it is hyperhq's official pod spec: UserPod.
its code is just wrongly put at runv. it will be moved back to hyperd.
UserPod will not be removed, any attempt will be rejected by xu. it is already an inuse user interface.